ceph-iscsi-config.spec

```diff
@@ -1,9 +1,25 @@
+#
+# spec file for package ceph-iscsi-config
+#
+# Copyright (C) 2017-2018 The Ceph iSCSI Config Project Developers. See
+# COPYING file at the top-level directory of this distribution and at
+# https://github.com/ceph-iscsi-config/ceph/blob/master/COPYING
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon.
+#
+# This file is under the GNU Lesser General Public License, version 3
```

Comment:
Shouldn't this use the same license as the source, rather than LGPL?

Reply:
@ddiss Good catch! Fixed.